tmpreaper.te

```diff
@@ -59,10 +59,7 @@ fs_delete_tmpfs_files(tmpreaper_t)
 files_read_var_lib_files(tmpreaper_t)
 files_purge_tmp(tmpreaper_t)
 files_delete_all_non_security_files(tmpreaper_t)
-# why does it need setattr?
-files_setattr_all_tmp_dirs(tmpreaper_t)
-files_setattr_isid_type_dirs(tmpreaper_t)
-files_setattr_usr_dirs(tmpreaper_t)
+allow tmpreaper_t non_security_file_type:dir { read setattr };
```

Comment:
I added commit 2673c5da26eab2f7c0a85f2d39522583a5144747, which contain interface for this rule. 
Could you change this line to: files_setattr_non_security_dirs(tmpreaper_t)? 
Then I'll merge this. 

Thank you!